src/PrestaShopBundle/Resources/config/routing/admin/improve/international/state.yml

```diff
@@ -1,6 +1,36 @@
 admin_country_states:
   path: /
   methods: [GET]
+  #condition needed to differentiate from admin_states_index path
+  condition: "request.isXmlHttpRequest()"
```

Comment:
i doubt condition is needed as I see its a different path in index.
```
admin_states_index:
  path: /list
```

But anyway I would suggest having index as `/` and smth like `/states` for ajax

Reply:
imlemented Julius suggestion